app/models/subnet.rb

```diff
@@ -187,7 +187,14 @@ def self.import(proxy)
     ProxyAPI::DHCP.new(:url => proxy.url).subnets.map do |s|
       # do not import existing networks.
       attrs = { :network => s["network"], :mask => s["netmask"] }
+
       next if first(:conditions => attrs)
+      if s["options"]
```

Comment:
Nitpick - What do you think abou extracting this handling of attrs to a separate method?